src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-inline: 16px;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: $image-width $image-height;
+    background-repeat: no-repeat;
+    width: $image-width;
+    height: $image-height;
+    margin-block: 32px 40px;
```

Comment:
don't mix margin-top and margin-bottom 
it could lead to margin collapse
choose one of them to add spacing between elements and be consistent